app/Models/Feed.php

```diff
@@ -481,10 +485,12 @@ function pubSubHubbubSubscribe($state) {
 				CURLOPT_FOLLOWLOCATION => true,
 				CURLOPT_RETURNTRANSFER => true,
 				CURLOPT_USERAGENT => 'FreshRSS/' . FRESHRSS_VERSION . ' (' . PHP_OS . '; ' . FRESHRSS_WEBSITE . ')',
-				CURLOPT_POSTFIELDS => 'hub.verify=sync'
-					. '&hub.mode=' . ($state ? 'subscribe' : 'unsubscribe')
-					. '&hub.topic=' . urlencode($url)
-					. '&hub.callback=' . urlencode($callbackUrl)
+				CURLOPT_POSTFIELDS => http_build_query(
+					'hub.verify' => 'sync',
+					'hub.mode' => ($state ? 'subscribe' : 'unsubscribe'),
+					'hub.topic' => urlencode($url),
```

Comment:
This broke PSHB. Fix on the way

Reply:
Should've been an array around it I suppose. Whoops. :)

Reply:
The array was the first issue (a bit higher up). I wonder why it was not caught by the linter when running Travis. Any idea?
This is another issue, with a double-encoding of hub.topic value :-)

Reply:
Should be fixed in https://github.com/FreshRSS/FreshRSS/pull/1643

Reply:
I'm not sure to what extent the linter picks up on things the regular parser will. Its main purpose is that which the parser won't pick up. (This particular error is all on me. The parser said it was a big bunch of string concatenation and I realized that PHP is a bit more efficient now than it once was... unfortunately introduced a slight type error in the process.)